scene/resources/box_shape_3d.cpp

```diff
@@ -29,6 +29,8 @@
 /**************************************************************************/
 
 #include "box_shape_3d.h"
+
+#include "primitive_meshes.h"
```

Comment:
```suggestion
#include "scene/resources/primitive_meshes.h"
```
Full paths are used here